lxd/instance/drivers/driver_qemu.go

```diff
@@ -7878,15 +7878,12 @@ func (d *qemu) checkFeatures(hostArch int, qemuPath string) (map[string]any, err
 		features["vhost_net"] = struct{}{}
 	}
 
-	vsockID, err := vsock.ContextID()
-	if err != nil || vsockID > 2147483647 {
-		// Fallback to the default ID for a host system if we're getting
-		// an error or are getting a clearly invalid value.
-		vsockID = 2
+	// Check if vhos-vsock is available
```

Comment:
This is already done in `qemu.Info()` which will only call `checkFeatures` if it exists. Not needed.